plugins/DuskVerb/tools/tuner/polish_only.py

```
@@ -0,0 +1,111 @@
+        print(f"    {k:24s} = {final[k]:.4f}")
+
+    final_dir = work / "final"
+    files = render(args.preset, final, args.vst3, final_dir)
+    if not files:
+        sys.exit("final render failed")
+    print(f"\nFinal render: {final_dir}")
+    cmd = [sys.executable, str(Path(__file__).resolve().parent / "full_check.py"),
+           str(final_dir), str(anchor_n.parent), "--name", args.preset]
+    if args.category:
+        cmd += ["--category", args.category]
+    subprocess.call(cmd)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)